src/codegen/utilities/interpreter_helpers.py

```diff
@@ -101,6 +101,20 @@ def get_interpreter_params(func):
     return (p for p in func.base_parameters if p.direction == "in")
 
 
+def get_grpc_interpreter_call_params(func, params):
+    """Gets the interpreter parameters for grpc request."""
+    compound_params = get_input_arguments_for_compound_params(func)
+    grpc_params = []
+    for param in params:
+        if param.parameter_name not in compound_params:
+            if param.is_enum:
+                grpc_params.append(f"{param.parameter_name}_raw={param.parameter_name}")
+            else:
+                grpc_params.append(f"{param.parameter_name}={param.parameter_name}")
+    grpc_params = sorted(list(set(grpc_params)))
```

Comment:
Sorry, I overlooked this earlier:
- Is deduplicating the parameters really necessary? Why are there duplicates?
- I would prefer to keep the parameters in the order defined by the metadata, not alphabetical order.

Reply:
This was added because the parameters were getting duplicated earlier. For example, `GetDigitalPowerUpStates` method had `ChannelName` twice in metadata and this was getting reflected in the request. Now that this has been fixed as part of a PR which was merged earlier, this sorting will be removed in https://github.com/ni/nidaqmx-python/pull/305